packages/identity-service/src/routes/relay.js

```diff
@@ -128,6 +123,23 @@ module.exports = function (app) {
               )
             }
           }
+
+          // Only reject relay for users explicitly blocked from relay and not for deactivations
+          if (blockAbuseOnRelay && user && user.isBlockedFromRelay) {
+            const decodedABI = libs.AudiusABIDecoder.decodeMethod(
+              txProps.contractRegistryKey,
+              txProps.encodedABI
+            )
+            const isDeactivation =
+              !user.is_deactivated &&
+              decodedABI.name === 'ManageEntity' &&
```

Comment:
this might have to be `manageEntity` and `args` should be `params` which is a list.


here's an example of a user update i pulled from staging
```
{
    "name": "manageEntity",
    "params": [
        {
            "name": "_userId",
            "value": "957015831",
            "type": "uint256"
        },
        {
            "name": "_entityType",
            "value": "User",
            "type": "string"
        },
        {
            "name": "_entityId",
            "value": "957015831",
            "type": "uint256"
        },
        {
            "name": "_action",
            "value": "Update",
            "type": "string"
        },
        {
            "name": "_metadata",
            "value": "{\"cid\":\"bagaaierakhyeykv2pfs67itowbc7vncka6yr6onbswbuans7o63hjvb2py7a\",\"data\":{\"is_verified\":true,\"is_deactivated\":false,\"name\":\"shouldnotverify\",\"handle\":\"shouldnotverify\",\"profile_picture\":null,\"profile_picture_sizes\":null,\"cover_photo\":null,\"cover_photo_sizes\":null,\"bio\":null,\"location\":\"Pasadena, CA\",\"artist_pick_track_id\":null,\"creator_node_endpoint\":null,\"associated_wallets\":null,\"associated_sol_wallets\":null,\"playlist_library\":{\"contents\":[{\"playlist_id\":555,\"type\":\"playlist\"}]},\"events\":{\"is_mobile_user\":true},\"allow_ai_attribution\":false,\"user_id\":957015831}}",
            "type": "string"
        },
        {
            "name": "_nonce",
            "value": "0xd7c3b090565ee121f8337a977eed317b62c02c8d82fdbed3f055ce1811494668",
            "type": "bytes32"
        },
        {
            "name": "_subjectSig",
            "value": "0xd8436aa2802e4ee2b00df08609691c43d71727887106dac94f1c5f6da2e5e5950cbb9402b5a68db1669819f83c70291fde7f1306e60fef09ccc58caad60162c31b",
            "type": "bytes"
        }
    ]
}
```